dockers/skylab/snss2-build-indices/modify_gtf.py

```diff
@@ -0,0 +1,129 @@
+import argparse
+import gzip
+import re
+
+def get_biotypes(biotypes_file_path):
```

Comment:
Add a comment about what the function is doing? Might be helpful for each of the functions if it's not too big a lift.

Reply:
Done.